src/astro/databases/postgres.py

```diff
@@ -26,3 +29,68 @@ def hook(self):
     def default_metadata(self) -> Metadata:
         # TODO: comment why
         return Metadata(database=self.hook.schema)
+
+    def merge_table(
+        self,
+        source_table: Table,
+        target_table: Table,
+        conflict_strategy: DBMergeConflictStrategy = "ignore",
+        merge_cols: Optional[List[str]] = None,
+        source_tables_cols: Optional[List[str]] = None,
+        target_tables_cols: Optional[List[str]] = None,
+    ) -> None:
+        """Merge two tables based on merge keys
+
+        :param source_table: Contains the rows to be appended to the target_table
+        :param target_table: Contains the destination table in which the rows will be appended
+        :param conflict_strategy: Action that needs to be taken in case there is a conflict
+        :param merge_cols: List of cols that are checked for uniqueness while merging,
```

Comment:
`conflict_columns` may be a more suitable name. I'd also advise using a list of Sqlalchemy `Column` instances - as we used in the `Table` definition, so we are consistent thought our interface about how we represent these columns.